sdk/go/common/resource/asset.go

```diff
@@ -940,7 +943,7 @@ func (a *Archive) Archive(format ArchiveFormat, w io.Writer) error {
 		if err != nil {
 			return err
 		}
-		_, err := io.Copy(w, ss)
+		_, err = io.Copy(w, ss)
```

Comment:
Missing size check?

```suggestion
		n, err = io.Copy(w, ss)
```

Reply:
(I am not sure if it's possible to find out what is the size of an `ArchiveFormat` though.)

Reply:
Yes, I think I started changing this then realized that there was no expected size to check against.